src/ucp/api/ucp_def.h

```diff
@@ -302,6 +324,32 @@ typedef struct ucp_err_handler {
 
 
 /**
+ * @ingroup UCP_EP
+ * @brief UCP address description in case of a client-server connection flow.
+ *
+ * This structure should be initialized in @ref ucp_ep_params_t
+ */
+typedef struct ucp_addr_sock_addr {
+    struct sockaddr   *sock_addr;           /**< Address in the form of a sockaddr */
```

Comment:
since the struct is already called "sock_addr", the internal fields names can be shortened to "addr" and "addrlen", same as the parameter names for `connect (2)`